torch/optim/_multi_tensor/adam.py

```diff
@@ -52,6 +52,18 @@ def __setstate__(self, state):
         for group in self.param_groups:
             group.setdefault('amsgrad', False)
 
+    def foreach_view_as_real(self,param):
```

Comment:
cc @ngimel should we make this an actual operator or it is fine to have it here?